.teamcity/builds/kotlinlang/buidTypes/BuildSitePages.kt

```diff
@@ -0,0 +1,208 @@
+package builds.kotlinlang.buidTypes
+
+import builds.apiReferences.kotlinx.coroutines.KotlinxCoroutinesBuildApiReference
+import builds.apiReferences.kotlinx.datetime.KotlinxDatetimeBuildApiReference
+import builds.apiReferences.kotlinx.serialization.KotlinxSerializationBuildApiReference
+import builds.kotlinlang.templates.DockerImageBuilder
+import jetbrains.buildServer.configs.kotlin.*
+import jetbrains.buildServer.configs.kotlin.buildSteps.ScriptBuildStep
+import jetbrains.buildServer.configs.kotlin.buildSteps.script
+
+const val kotlinWebsiteSetup = "/kotlin-website-setup.sh"
+
+object BuildSitePages : BuildType({
+  name = "Build site pages"
+
+  templates(DockerImageBuilder)
+
+  artifactRules = """
+        dist/** => pages.zip
+        robots.txt => pages.zip
+    """.trimIndent()
+
+  vcs {
+    root(vcsRoots.KotlinLangOrg)
+    cleanCheckout = true
+  }
+
+/*  triggers {
+    vcs {
+      branchFilter = """
+          +:<default>
+          +:spring-learning-path
+      """.trimIndent()
+    }
+  }*/
+
+  steps {
+    script {
+      name = "Build html pages"
+      scriptContent = """
+                #!/bin/bash
+                
+                set -x
+                
+                cat $kotlinWebsiteSetup
```

Comment:
O_o

Reply:
It just works as a variable because it is a Kotlin file, no magic here :)